tcms/testruns/tests/test_admin.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+
+from http import HTTPStatus
+
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+from tcms.tests import LoggedInTestCase
+from tcms.tests.factories import UserFactory
+
+
+class TestSuperuserActions(LoggedInTestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.superuser = UserFactory()
+        cls.superuser.is_staff = True
+        cls.superuser.is_superuser = True
+        cls.superuser.set_password('password')
+        cls.superuser.save()
+
+        cls.regular_user = UserFactory()
+        cls.regular_user.is_staff = True
+        cls.regular_user.set_password('password')
+        cls.regular_user.save()
+
+        cls.url = reverse('admin:auth_user_delete', args=[cls.regular_user.pk])
+
+    def test_superuser_should_be_able_to_delete_any_user(self):
+        self.client.login(  # nosec:B106:hardcoded_password_funcarg
+            username=self.superuser.username,
+            password='password')
+        response = self.client.get(self.url)
+
+        # verify there's the Yes, I'm certain button
+        self.assertEqual(HTTPStatus.OK, response.status_code)
+        self.assertContains(response, _("Yes, I'm sure"))
+        response = self.client.post(self.url, {'post': 'yes'}, follow=True)
+        self.assertEqual(HTTPStatus.OK, response.status_code)
+        self.assertNotContains(response, '<a href="/admin/auth/user/%d/change/">%s</a>' %
+                               (self.regular_user.pk, self.regular_user.username), html=True)
+
+    def test_super_user_can_add_view(self):
```

Comment:
You are testing with a regular user so name must reflect this.

You first testing scenario is:
1) Login as a regular user
2) Try adding a new TR via the admin interface (Add button)
3) Expected result is that user is redirected back to the changelist page.

This is what the source code below tries to model so naming must match.